apps/odyssey-stats/src/styles/variables.scss

```diff
@@ -0,0 +1,13 @@
+@import "@automattic/components/src/styles/typography.scss";
+
+$display-type-font: -apple-system,BlinkMacSystemFont,"Segoe UI","Roboto","Oxygen-Sans","Ubuntu","Cantarell","Helvetica Neue",sans-serif;
```

Comment:
Could we use [$sans](https://github.com/Automattic/wp-calypso/blob/trunk/packages/typography/styles/variables.scss#L10) or [$a8c-font-family-sans](https://github.com/Automattic/wp-calypso/blob/trunk/packages/components/src/styles/_typography.scss#L1) for this variable?

Reply:
Good catch, fixed!